Core/include/Acts/Seeding/InternalSpacePoint.hpp

```diff
@@ -83,6 +118,16 @@ inline InternalSpacePoint<SpacePoint>::InternalSpacePoint(
   m_r = std::sqrt(m_x * m_x + m_y * m_y);
   m_varianceR = variance.x();
   m_varianceZ = variance.y();
+
+  m_doubleMeasurement.topHalfStripLength = sp.topHalfStripLength();
```

Comment:
Was wondering about these lines... you read directly the following quantities from the space point sp.
Maybe we can use the same approach as done for the global position and the variance using a std::function that is passed to the BinnedSPGroup to extract the additional details from the space point:

https://github.com/acts-project/acts/blob/db7f497303712fff32cf418646737a205a6e1adc/Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp#L140-L158

Of course we need a way to save and return the quantities.